source/gcap/introduction.rst

```diff
@@ -0,0 +1,40 @@
+简介
+====
+
+gCAP 可用于反演震源机制解。
+
+功能
+----
+
+1. 支持反演完整的地震矩张量
+2. 支持使用三维模型下的格林函数
+
+命令及脚本
+----------
+
+gCAP 中提供了如下几个命令和脚本。
+
+- ``cap3D``\ ：用于反演地震矩张量的主程序，也是 gCAP 的核心程序
+- ``cap3D.pl``\ ：对 ``cap3D`` 的封装，一般情况下直接使用该脚本即可
+- ``depth.pl``\ ：用于读取 ``cap3D`` 输出的不同深度震源的结果，并估计最佳震源深度
+- ``mtdcmp``\ ：将地震矩张量分解为 ISO、DC 以及 CLVD 三个张量
+- ``radpttn``\ ：将 ISO、DC 以及 CLVD 三个张量合成为归一化的地震矩张量，
+  并计算 P、SV 以及 SH 波的辐射花样
+
+引用
```

Comment:
Do we want to explain the differences between cap, gcap, gcap3d, cap-tele, gcap-tele, gcap-sf et al.?

Reply:
Yes, but I think should be put on a separate page after we finish gCAP3D first.
This topic deserves more details

Reply:
OK